packages/api/src/User/User.resolver.ts

```diff
@@ -48,7 +64,9 @@ export class UserResolver {
 
   @ResolveField(() => UserPreferences)
   async userPreferences(@Parent() user: User) {
-    const userPreferences = await this.userPreferencesService.findByUser(user.id);
+    const userPreferences = await this.userPreferencesService.findByUser(
+      user.id
+    );
     return userPreferences;
```

Comment:
Small thing, but you can just combine these into one line. EG: `return await this.UserPref....`

Reply:
Hey @tristanMatthias, code has been refactored.